src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,19 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private double loadCapacity;
+    private int numberOfAxles;
```

Comment:
Don't forget about access modifiers for your custom fields in `Truck` class. You should use private access modifier for `numberOfAxles` to encapsulate the field properly.